.travis.yml

```diff
@@ -1,13 +1,31 @@
 language: perl
+# Archive::Zip currently requires sudo
+sudo: required
```

Comment:
Some dependencies on CPAN are in .zip files. Archive::Zip is needed to extract them, but Archive::Zip is currently broken in travis containers. See its RT